treerec/implementation.md

```diff
@@ -5,10 +5,6 @@
 Both individuals and nodes (= genomes) have flags. We use individual flags for
 internal bookkeeping in SLiM; while msprime uses the SAMPLE flag of nodes.
 
-0. The first generation individuals get entered into the individual table, with
-    the `INDIVIDUAL_FIRST_GEN` flag, so we know to keep these around through simplify.
-    Their genomes' node IDs are also added to `remembered_genomes_`.
-
 1. Every new individuals' genomes each get the `NODE_IS_SAMPLE` flag,
```

Comment:
This wording suggests that every individual is added, whereas I think that (at the moment) only "remembered" and "alive" are entered. Should we say "If an individual is added, it initially gets the `NODE_IS_SAMPLE` flag, ..."